src/mux_client_impl/mod.rs

```diff
@@ -0,0 +1,114 @@
+#![allow(
+    missing_docs,
+    missing_debug_implementations,
+    rustdoc::broken_intra_doc_links,
+    unreachable_pub
+)]
```

Comment:
I think all of these allows should be removed.

Reply:
Removed.